### PR TITLE
Handle Clang versions dynamically

### PR DESCRIPTION
**Pull request by rs--**

### CODE REVIEW

File: src/jenkins/common/Openenclave.groovy
Comment (CyanDevs) [inline src/jenkins/common/Openenclave.groovy:85]: clang-7 case would be defunct since it would never happen?

**Review by CyanDevs — APPROVED** [review]

**Comment by ghost** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/openenclave/openenclave-ci/pull/30#pullrequestreview-664946960" title="Approved">CyanDevs</a>*, *<a href="https://github.com/openenclave/openenclave-ci/pull/30#" title="Author self-approved">rs--</a>*

The full list of commands accepted by this bot can be found [here](?repo=openenclave%2Fopenenclave-ci).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/openenclave/openenclave-ci/blob/master/OWNERS)~~ [rs--]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
